examples/stable_diffusion_xl/tools/model_conversion/convert_diffusers_to_mindone_sdxl.py

```diff
@@ -0,0 +1,409 @@
+# Script step 1:
+# for converting a HF Diffusers saved pipeline to a Stable Diffusion checkpoint.
+# *Only* converts the UNet, VAE, and Text Encoder.
+# Does not convert optimizer state or any other thing.
+
+# Script step 2:
+# convert Stable Diffusion checkpoint(torch) to MindOne Stable Diffusion checkpoint(mindspore)
+# *Only* converts the UNet, VAE, and Text Encoder as above
+
+# Script step 3：
+# insert Stable Diffusion checkpoint(mindspore) to sd_xl_base_1.0_ms.ckpt, then get a new mindspore checkpoint
```

Comment:
Please put these instructions on how to convert to a README file

Reply:
okay, I'll add it right away

Reply:
Done